Resources/Locale/en-US/store/uplink-catalog.ftl

```diff
@@ -142,6 +142,9 @@ uplink-gatfruit-seeds-desc = And who says guns don't grow on trees?
 uplink-necronomicon-name = Necronomicon
 uplink-necronomicon-desc = An unholy book capable of summoning a demonic familiar.
 
+uplink-hot-potato-name = Hot Potato
+uplink-hot-potato-desc = When activated can only be transferred away by hitting another being with it!
```

Comment:
the wording of this is sus
suggest using `Once activated, this time bomb can't be dropped - only passed to someone else!"

Reply:
ok